### ISSUE TITLE
Name Order issue

### ISSUE THREAD

**Issue by inouekoichi** [roundcube/roundcubemail#6160]
Hi,
Addressbook list name order is superseded by fullname if the contact contains the fullname entry.   But, when I select name order to be 'surname firstname' (which is usual order for Japanese/Chinese/Korean name) in the preference pane, the order is used prior to fullname.   I think fullname should be used because addressbook contains both order contacts.

**Comment by alecpl** [comment]
I don't fully understand. Could you give some examples?

**Comment by inouekoichi** [comment]
I selected 'surname firstname' order in the prefernce pane.   Then address list shows like this:

Tanaka Taro    ...(OK)
Inoue Koichi    ...(OK)
Brown John    ...(NG)
Smith John    ...(NG)

(Tanaka and Inoue are Japanese popular familyname, and my real address book actually uses utf-8 kanji characters: 田中(Tanaka) 太郎(Taro))

I made 'fullname' entry for the card of John Brown and John Smith, 
FN: John Brown
FN: John Smith
But these are not used for the list of addresses.   When I selected 'fullname' in the preference and make 'fullname' entry for Tanaka like this:
FN: Tanaka Taro
and erase FN entry for Brown and Smith, then the address list is:

Tanaka Taro    ...(OK)
Koichi Inoue    ...(NG)
John Brown    ...(OK)
John Smith    ...(OK)

My real address book contains many Japanese peoples' cards and few foreigners'.  So I want to make fullname entries only for the few foreigners.   But 'surname firstname' order lists all names in this order.

....and, in the address card, names are always shown like this:

Taro     Tanaka

Koichi    Inoue

John   Brown

John   Smith

this order is not affected by the selection in the preference pane.   I think it is a better solution that fullname entries are always used for address list, and names are shown like

Tanaka   Taro

Inoue   Koichi

Brown   John
(John Brown)

Smith   John
(John Smith)

in the addrss card, when I select 'surname firstname' order in the preference pane.
